.github/workflows/simulate-merge-pr.yml

```
@@ -0,0 +1,95 @@
+name: Simulate merging a PR 
+
+# Trigger when PR created, when new commits added to it (synchronize), or when the PR title has been modified since the PR title changes behavior of a simulated merge 
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
```

This is only relevant when [creating a PR against](https://github.com/customerio/customerio-ios/blob/beta/.github/workflows/deploy-sdk.yml#L5) `main` and `v1` right?

Could we do this?

```suggestion
  pull_request:
    types: [opened, edited, synchronize]
    branches:
      - main
      - v1
```